go-controller/pkg/node/openflow_manager.go

```diff
@@ -338,3 +327,25 @@ func bootstrapOVSFlows() error {
 
 	return nil
 }
+
+// localnetInfo returns the patch port name and the bridge name for the default network
+func localnetInfo(portsOutput string) (string, string) {
+	var bridge string
+	var patchPort string
+	// patch-br-int-to-<bridge name>_<node>
+	r := regexp.MustCompile("^patch-(.*)_.*?-to-br-int$")
+	for _, line := range strings.Split(portsOutput, "\n") {
+		matches := r.FindStringSubmatch(line)
+		if len(matches) == 2 {
+			patchPort = matches[0]
+			bridge = matches[1]
+			if strings.Contains(bridge, "ovn_localnet") {
```

Comment:
maybe if we change the capture on the regex above to be non-greedy I can match against the `ovn_localnet_port` constant here. 

I didn't want to update it, but it maybe provides better readability. 

Thoughts ?

Reply:
This is half non-sense. I think it is better now.